### PR TITLE
Create recommendations from Kelly for Benicia

### LINKED ISSUE #8: Create a file for kellyrwds' hometown (Benicia, CA)
# Benicia, CA 🙌
- [x] Create a branch
- [x] Add commits (make changes)
- [x] Open a Pull Request
- [x] Collaborate
- [ ] Seek approval
- [ ] Merge the changes (or PR pull request)

### PR DESCRIPTION
**Pull request by kellyrwds**
closes #8

### CODE REVIEW

File: places/kellyrwds-benicia.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Benicia, CA
+
+### Places to Eat
+*Pizza Pirate*
+### Places to Avoid
+*That Greek place I forgot the name*
+### Places to Go
+*Walk down Main Street*
+### Fun Facts
+*Benicia was once the state capital*
```
Comment (lamberp6) [inline places/kellyrwds-benicia.md:11]: Did not know that!

File: places/kellyrwds-benicia.md [hunk 2]
```diff
@@ -7,4 +7,4 @@
 ### Places to Go
 *Old Capital Building*
 ### Fun Facts
-*Benicia was once the state capital for CA*
+*Benicia was once the state capital*
```
Comment (kaz-gene-com) [inline places/kellyrwds-benicia.md:11]: The Ace hardware has great deals on cheap tools!

File: places/kellyrwds-benicia.md (hunk 1 above)
Comment (kellyrwds) [reply places/kellyrwds-benicia.md:11]: Yes! @lamberp6 You can visit the small state capitol building. Still standing.

**Review by kaz-gene-com — APPROVED** [review]
Review summary just to see where it goes!

**Pull request merged**